_posts/2016-01-30-iOS-Junior-Interviews.markdown

```diff
@@ -295,41 +296,121 @@ This bit is hard, because you're in limbo, and it can take a while. I think with
 
-If you're not active on Twitter, you should start trying. Every time you think of something worth saying to someone next to you, say it to them, then say it to twitter. Show off pictures of what you're working on, reply to people with more followers than you when they ask questions, talk about code. It took me 4 years to hit my first thousand followers, hopefully you can be more interesting [than I was](https://twitter.com/orta/status/13912101) for a while.
+If you're not active on Twitter, you should start trying. Every time you think of something worth saying to someone next to you, say it to them, then say it to twitter. Show off pictures of what you're working on, reply to people with more followers than you when they ask questions, talk about code. It took me 4 years to hit my first thousand followers, hopefully you can be more interesting [than I was][57] for a while.
 
-This article covers [how developers in general use twitter](http://blog.leif.me/2013/11/how-software-developers-use-twitter/) it's an interesting read. You can get a lot of value by following developers whose work you rely on, or whose [blog posts you think are awesome](https://twitter.com/orta).
+This article covers [how developers in general use twitter][58] it's an interesting read. You can get a lot of value by following developers whose work you rely on, or whose [blog posts you think are awesome][59].
 
-When people say GitHub is your resume, they are right and they are wrong. GitHub can help get you through the door, but you can't rely on that in your interviews. Check out the advice at the [end of this post](http://code.dblock.org/2011/07/14/github-is-your-new-resume.html) on how to make your GitHub look good for people looking at whether to interview you.
+When people say GitHub is your resume, they are right and they are wrong. GitHub can help get you through the door, but you can't rely on that in your interviews. Check out the advice at the [end of this post][60] on how to make your GitHub look good for people looking at whether to interview you.
 
-However you can [distinguish yourself by contributing to Open Source](https://ma.tt/2016/02/getting-a-job-after-coding-bootcamp/). [Fix READMEs](https://github.com/alloy/lowdown/pull/8), [improve documentation ](https://github.com/supermarin/ObjectiveSugar/pull/20) or [the Pods you use](http://artsy.github.io/blog/2016/01/28/being-a-good-open-source-citizen/). These slides go [into other ways](https://speakerdeck.com/orta/contributing-to-open-source-doesnt-require-changing-the-world) to contribute."
+However you can [distinguish yourself by contributing to Open Source][61]. [Fix READMEs][62], [improve documentation ][63] or [the Pods you use][64]. These slides go [into other ways][65] to contribute."
```

Comment:
Another tip I'd offer is participating on Stack Overflow. Aside from being useful for getting out of jams, it's a great way to help other devs out and to learn.